modules/grandpa/src/benchmarking.rs

```diff
@@ -66,6 +67,10 @@ const MAX_VOTE_ANCESTRIES: u32 = 1000;
 // number of validators.
 const MAX_VALIDATOR_SET_SIZE: u32 = 1024;
 
+fn header_number<T: Config<I>, I: 'static, R: From<u32>>() -> R {
```

Comment:
Does `R` stand for something clever here? I would've expected `N` for number, lol

Reply:
it's for "return value", fine to change it to `N` :)